mockserver/generate_mock_data.js

```diff
@@ -26,14 +26,12 @@ const generateMockData = async () => {
         `${CORE_URL}/state/${agg}`,
         `./public/${API_VERSION}/state/${agg}.json`
       );
-      if (agg !== 'last_hour') {
-        detailsZones.forEach(async (zoneId) => {
-          await fetchAndStoreData(
-            `${CORE_URL}/details/${agg}/${zoneId}`,
-            `./public/${API_VERSION}/details/${agg}/${zoneId}.json`
-          );
-        });
-      }
+      detailsZones.forEach(async (zoneId) => {
+        await fetchAndStoreData(
+          `${CORE_URL}/details/${agg}/${zoneId}`,
+          `./public/${API_VERSION}/details/${agg}/${zoneId}.json`
+        );
+      });
```

Comment:
The only changes in this block of code are the removal of the if statement and the de-indentation of its code block.